services/property-svc/internal/property-value/queries/v1/get_property_values_by_subject_id.go

```diff
@@ -0,0 +1,50 @@
+package v1
+
+import (
+	"context"
+	pb "gen/proto/services/property_svc/v1"
+	"github.com/google/uuid"
+	"hwdb"
+	"hwutil"
+	"property-svc/internal/property-value/models"
+	"property-svc/repos/property_value_repo"
+)
+
+type GetPropertyValuesWithPropertiesBySubjectIQueryHandler func(ctx context.Context, subjectID uuid.UUID) ([]models.PropertyValueWithProperty, error)
+
+func NewGetPropertyValuesWithPropertiesBySubjectIDQueryHandler(propertyValueRepo *property_value_repo.Queries) GetPropertyValuesWithPropertiesBySubjectIQueryHandler {
+	return func(ctx context.Context, subjectID uuid.UUID) ([]models.PropertyValueWithProperty, error) {
+
+		propertyValuesWithProperties, err := hwdb.Optional(propertyValueRepo.GetPropertyValuesWithPropertyBySubjectID)(ctx, subjectID)
```

Comment:
[sqlc.yaml](https://github.com/helpwave/services/blob/main/services/property-svc/sqlc.yaml#L13) Optional is not needed